styles.css

```diff
@@ -272,4 +272,7 @@ body:not(:has(.ddoc)) {
       background-color: var(--color-neutral-muted) !important;
     }
   }
+  .markdown_summary :not(pre) > code {
+    background-color: var(--color-neutral-muted) !important;
```

Comment:
this is unrelated to the single col thing I just noticed that the style above didn't catch all code blocks